docs_user/modules/proc_rolling-back-the-control-plane-adoption.adoc

```diff
@@ -2,27 +2,19 @@
 
 = Rolling back the control plane adoption
 
-If you encountered a problem during the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services that prevents you from completing the adoption procedure, you can roll back the control plane adoption.
+If you encountered a problem that prevents you from completing the adoption of the {rhos_prev_long} ({OpenStackShort}) control plane services, you can roll back the control plane adoption.
 
 [IMPORTANT]
-The roll back operation is only possible during the
-control plane parts of the adoption procedure. If you altered the data plane
-nodes in any way during the procedure, the roll back is not possible.
-
-During the control plane adoption, services on the source cloud's
-control plane are stopped but not removed. The databases on the source
-control plane are not edited by the adoption procedure. The
-destination control plane received a copy of the original
-control plane databases. The roll back procedure assumes that the data
-plane has not yet been touched by the adoption procedure and it is
-still connected to the source control plane.
+You can only roll back the control plane adoption during the parts of the adoption procedure that alter the control plane. If you altered the data plane
+nodes in any way, you cannot perform the rollback procedure.
+
+During the control plane adoption, services on the {OpenStackShort} control plane are stopped but not removed. The databases on the {OpenStackShort} control plane are not edited during the adoption procedure. The {rhos_long} control plane receives a copy of the original control plane databases. The rollback procedure assumes that the data plane has not yet been modified by the adoption procedure, and it is still connected to the {OpenStackShort} control plane.
```

Comment:
I guess this line is explaining the previous line. Does it mean you can't rollback (won't be able to do it) if data plane is altered, or you shouldn't rollback (if you do it, you'll break stuff)?

If it's the latter, maybe the previous line could be something like: "Do not attempt to rollback if you altered the data plane nodes in any way." Could double check with SME about this or what the potential issues are - data corruption maybe?

Reply:
I don't think it's possible to roll back if data plane nodes are altered. It's that they can't, not that they shouldn't. But @jistr can you confirm?

Reply:
Technically they could attempt the rollback but likely it would break stuff. So +1 on "Do not attempt to rollback if you altered the data plane nodes in any way."